prepare.sh

```diff
@@ -5,4 +5,7 @@ set -e
 echo "Stopping all containers"
 rce stop $(rce ps -a -q)
 
+echo "Stop cron jobs"
+/etc/init.d/crond stop
```

Comment:
This doesn't exist in newer OS versions and will cause this script to fail due to `set -e`